frontend/src/component/operator-wizard/page-component/Page.js

```diff
@@ -242,9 +250,28 @@ export default class Page extends Component {
     return jsonObject;
   }
 
+  addObjectFields(field, jsonObject) {
+    //  let childJson ={};
+    if (Array.isArray(field.fields)) {
+      field.fields.forEach(field => {
+        if (field.type === "object") {
+          jsonObject = this.addObjectFields(field, jsonObject);
+        } else {
+          const value = field.type === "checkbox" ? field.checked : field.value;
+          if (value !== undefined) {
+            let jsonPath = this.getJsonSchemaPathForYaml(field.jsonPath);
+
+            jsonObject[jsonPath] = value;
+          }
+        }
+      });
+    }
+    return jsonObject;
+  }
+
   createSampleYaml() {
     var sampleYaml = {};
-    sampleYaml = this.createSampleYamlfromForm(sampleYaml);
+    // sampleYaml = this.createSampleYamlfromForm(sampleYaml);
     sampleYaml = this.createSampleYamlfromPages(sampleYaml);
     return sampleYaml;
```

Comment:
you can directly return the `this.createSampleYamlfromPages(sampleYaml);` without defining any var